src/dotnet/APIView/APIViewTest/FieldTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using System.Collections.Immutable;
+using System.Linq;
+using APIView;
+using Xunit;
+
+namespace APIViewTest
+{
+    public class FieldTests
+    {
+        [Fact]
+        public void FieldTestReadOnly()
+        {
+            var reference = MetadataReference.CreateFromFile("TestLibrary.dll");
```

Comment:
Add a base type for tests and add a GetTestMember(string typeName, string memberName) method to it. You would be able to remove a lot of duplicated code from your tests this way.